tools/scylla-nodetool.cc

```diff
@@ -4297,6 +4364,38 @@ For more information, see: {}"
                 table_stats_operation
             }
         },
+        {
+            {
+                "tablet-repair",
+                "Synchronize tabelts-based data between nodes in the background",
+fmt::format(R"(
+When running nodetool tablet-repair on a single node, all tablets of
+a specified keyspace(s) are repaired, even if there is no replica on
+this node.
+
+To repair all of the vnodes-based data in the cluster, it is enough to run
+nodetool tablet-repair on one node only.
```

Comment:
First part of the paragraphs talks about vnodes, you can just drop the "vnodes-based" bit.